packages/cardano-graphql/test/util.test.ts

```diff
@@ -28,4 +28,21 @@ describe('util', () => {
       await expect(provider.fn()).rejects.toThrowError(new ProviderError(ProviderFailure.Unknown, error));
     });
   });
+
+  describe('getExactlyOneObject', () => {
```

Comment:
Please add at least one happy path test to help communicate the purpose of this function